src/Magento/FunctionalTestingFramework/Test/Util/ActionMergeUtil.php

```diff
@@ -71,6 +78,25 @@ public function __construct($contextName, $contextType)
     {
         $this->name = $contextName;
         $this->type = $contextType;
+
+        # This list is sorted alphabetically. Please maintain that order when modifying.
+        # They array_flip() is a trick to speed up checking for items in the blacklist.
+        $this->waitsBlacklist = array_flip([
+            "assertArrayHasKey", "assertArrayIsSorted", "assertArrayNotHasKey", "assertArraySubset", "assertContains",
```

Comment:
This list is not comprehensive. There are probably more actions to be added. But I was getting failed build results.
So to compensate for that, I've reduced the initial list and we can add more later if needed.